### DISCUSSION TITLE
Blur effect on Bottom Tab Navigator?

### DISCUSSION THREAD

**Discussion by rafaelrcamargo** [react-navigation/react-navigation#11195] [Q&A]
Hello,

I am currently working on a react-native project and I am trying to add a blur effect on the bottom tab navigator. However, I am facing some difficulties in implementing it. I would greatly appreciate if someone could provide some guidance or tips on how to achieve this.

Thank you for your time and help!

Best regards,
Rafael.

**Comment by k2on** [comment]
```jsx
        <Tabs.Navigator screenOptions={{
            tabBarShowLabel: false,
            tabBarActiveTintColor: "black",
            tabBarStyle: { position: "absolute", bottom: 0 },
            tabBarBackground: () => <View className="bg-none absolute bottom-0 h-full w-full">
                <BlurView
                    style={{ height: 100 }}
                    blurType="light"
                    blurAmount={20}
                  ></BlurView>
            </View>,
```

**Reply by sreeju19kpz** [reply]
use overflow hidden on tabbar styles incase if a blur layer appears on your entire screen
